troposphere/settings/__init__.py

```diff
@@ -1,4 +1,5 @@
+from troposphere.settings.default import *
 try:
-    import troposphere.settings.local
+    from troposphere.settings.local import *
```

Comment:
@kamfonik  Sorry, I should have mentioned that. But you were right about we need to refactor it. That's why it got changed before. Just need to come up a better way. Revert it for now.